.travis.yml

```diff
@@ -1,14 +1,12 @@
 language: node_js
-matrix:
-  include:
-    - node_js: "6.9.1"
-    - node_js: "7"
-    - node_js: "8"
 dist: trusty
 env:
   global:
     - ARTIFACTS_AWS_REGION=us-west-2
     - ARTIFACTS_S3_BUCKET=lighthouse-screenshots-debugging
```

Comment:
we can totally nix that screenshot saving too btw if we want some extra wins :)

Reply:
I only install it when we want to send it over so not a big issue.